UnrarKit.podspec

```diff
@@ -95,7 +98,8 @@ Pod::Spec.new do |s|
                         "Libraries/unrar/win32stm.cpp"
     ss.pod_target_xcconfig = { "OTHER_LDFLAGS" => "$(inherited) -lc++",
                                "OTHER_CFLAGS" => "$(inherited) -Wno-return-type -Wno-logical-op-parentheses -Wno-conversion -Wno-parentheses -Wno-unused-function -Wno-unused-variable -Wno-switch -Wno-unused-command-line-argument",
-                               "OTHER_CPLUSPLUSFLAGS" => "$(inherited) -DSILENT -DRARDLL $(OTHER_CFLAGS)" }
+                               "OTHER_CPLUSPLUSFLAGS" => "$(inherited) -DSILENT -DRARDLL $(OTHER_CFLAGS)",
+                               "USER_HEADER_SEARCH_PATHS" => "${PODS_ROOT}/UnrarKit/Libraries/unrar" }
     ss.compiler_flags = "-Xanalyzer -analyzer-disable-all-checks"
```

Comment:
Below this line, add `ss.library = "z"`, so it can find the `crc32` function. I'm not sure why it was able to find it before without this.